lms/djangoapps/certificates/views/webview.py

```diff
@@ -104,48 +95,7 @@ def _update_certificate_context(context, course, user, user_certificate):
         context['organization_short_name'] = partner_short_name
         context['organization_logo'] = organization.get('logo', None)
```

Comment:
can we also move the organization related information into separate method ? e.g. `_update_course_organization_info`